src/platforms/common/configuration/options.mdx

```diff
@@ -372,6 +372,12 @@ _(New in version 6.0.0)_
 
 </ConfigKey>
 
+<ConfigKey name="attach-screenshot" supported={["apple.ios"]}>
```

Comment:
This key already exists, L367, adapt the text if necessary

Reply:
Can you provide a suggestions on how to do different text for different platform? 
Thanks!!

Reply:
You can just amend that hard crashes are not supported on iOS and this is a best-effort, the common content for docs also states that and it should be available for Android and iOS.